tests-e2e/step-definitions/standard-ui/form.js

```diff
@@ -69,9 +69,9 @@ When('I set Input field with id {string} to {string}',
     client.useCss();
 
     await client.expect.element(`#${id}`).to.be.visible;
-    await betterClearValue(`#${id}`);
```

Comment:
Is this working for all tests? I thought i have in mind, that in some cases, the normal `clearValue` will not correctly clear vue models. But if all tests are fine, should be okay :)

Reply:
I'm using the nightwatch method `clearValue()` instead and it seems to work fine. I don't really understand why we needed an own function to begin with.